draft-ietf-quic-http.md

```diff
@@ -1394,6 +1394,12 @@ the entire connection when an error is encountered.  These are referred to as
 {{QUIC-TRANSPORT}}.  An endpoint MAY choose to treat a stream error as a
 connection error.
 
+Because new error codes can be defined without negotiation (see {{extensions}}),
+receipt of an unknown error code or use of an error code in an unexpected
+context MUST NOT be treated as an error.  However, the fact that a stream closed
```

Comment:
This last sentence is a bit odd.  How about something like "However, closing a stream can constitute an error regardless of the error code -- see..."